package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "leaderboard_01",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "build": "webpack"
```

Comment:
Currently, I think you don't have any script to run or start your server. hence, it is not possible to run your project on a server🥲. Kindly add the necessary script to make it possible to run your project on the server and ensure it is possible to run your project on a local server👍.